be/src/Project/Services/Client/Client.Api/v1/RequestModels/ClientRequestModel.cs

```diff
@@ -10,5 +11,6 @@ public class ClientRequestModel : RequestModelBase
         public string CommunicationLanguage { get; set; }
         public string PhoneNumber { get; set; }
         public Guid? OrganisationId { get; set; }
+        public IEnumerable<Guid> Groups { get; set; }
```

Comment:
**suggestion:** Rename to ClientGroupIds and everywhere where you use list of Guids